htmlbook-xsl/xspec/xrefgen.xspec

```diff
@@ -1657,5 +1657,114 @@ and the script from https://github.com/xspec/xspec.">
     </x:call>
     <x:expect label="Trimmed URL should be returned as is">oreilly.com</x:expect>
   </x:scenario>
-  
+
+<!-- TOOLSREQ-6431 XREFS for Admonitions begins -->

```

Comment:
Ticket numbers in comments: are they useful? :) I can't think of a concrete problem with them, I just know I end up deleting them a lot in our other codebases. The ticket reference might allow us to figure out who wanted this change later, but it seems like the tests themselves explain pretty well the intention of the change (or maybe, the tests _should_).

Reply:
That's a good point. Considering ticket numbers are not used elsewhere in the XSPEC codebase I went ahead and removed it.